pylintrc

```diff
@@ -3,10 +3,10 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
-extension-pkg-whitelist=orjson
+extension-pkg-whitelist=
 
 # Specify a score threshold to be exceeded before program exits with error.
-fail-under=10.0
+fail-under=8.3
```

Comment:
Are these changes to the pylint config actually related to fixing #55? They feel like part of a different/bigger issue. Also, it feels like we should leave the threshold set at 10.0 and fix the problems that cause failures.

Reply:
No, I just noticed this and figured a quick improvement would be nice. See the commit message for more thoughts on this; of course it'd be better to fix the problems, however, it's potentially a lot of work and in the meantime we have those checks disabled across the board, which makes further quality degradation (under those disabled checks) invisible. So I thought it's better to reenable them and lower the threshold until we fix them. I think that's the use case this setting is intended for.

Reply:
Ok. I guess I don't really like the whole concept of the threshold. It feels weird to me to allow the code to have a general level of meh-ness, rather than being specific about what kinds of shortcomings are acceptable.

I agree that having the checks disabled everywhere isn't good, though.

Reply:
> It feels weird to me to allow the code to have a general level of meh-ness, rather than being specific about what kinds of shortcomings are acceptable.

I agree, but allowing specific meh-ness to proliferate is IMO strictly worse ;)